src/jsoo/dune

```diff
@@ -0,0 +1,4 @@
+(library
+ (name maki_jsoo)
+ (public_name maki.jsoo)
```

Comment:
I think it should be `maki_jsoo` or `maki-jsoo` here as well, since it's a separate library that has a distinct name (and I agree with this choice! Dependencies should be explicit!)